config/manager/role.yaml

```diff
@@ -5,7 +5,9 @@ metadata:
   creationTimestamp: null
   name: kube-trigger-manager-role
 rules:
-- resources:
+- apiGroups:
```

Comment:
I think this file is auto generated, can you help to check that is there anything wrong with the generator? cc @charlie0129

Reply:
Yes, that's right. Maybe newer versions of kubectl require apiGroups to be set.

I fixed the generator to generate that field.